fedimint-api/src/module/mod.rs

```diff
@@ -425,8 +425,7 @@ where
 
 #[async_trait]
 pub trait ServerModule: Debug + Sized {
-    const KIND: ModuleKind;
-
+    type Gen: ModuleGen;
```

Comment:
Now I'm unsure which way around this reference makes the most sense … but I think this should work (just changes if you attach modules or module gens to the federation builder eventually).

Reply:
Might be both.